### PR TITLE
Better logging from MigrateAssessmentsJob, clobber, dont create Exit Assessments for open enrollments

### PR DESCRIPTION
**Pull request by gigxz**

### CODE REVIEW

File: drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb [hunk 1]
```diff
@@ -141,8 +163,17 @@ def build_assessments(enrollment_scope:, data_collection_stages:, unique_by_info
 
         # Build FormProcessor with IDs to all related records
         assessment.build_form_processor(**value)
-        assessment.save!
+
+        if assessment.exit? && value[:exit_id].nil?
+          # There appear to be lots of "exit" data-collection-stage records for enrollments that don't have an Exit record. Skip those
+          Rails.logger.info "Skipping Exit Assessment for open enrollment. EnrollmentID: #{assessment.enrollment_id}"
+          skipped_exit_assessments += 1
```
Comment (gigxz) [inline drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb:170]: @eanders I believe this is likely an issue where a client was exited and then un-exited. The data associated with the Exit Data Collection Stage remains, but the Exit record does not. Or, maybe there's something else going on around Exit records.

In this case –– like in the case where there are duplicate records for a given assessments –– I think we should delete those records. I'm holding off on that right now until I get confirmation.

File: drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb [hunk 2]
```diff
@@ -7,8 +7,12 @@
 module Hmis
   class MigrateAssessmentsJob < BaseJob
     include Hmis::Concerns::HmisArelHelper
+    include NotifierConfig
+
     attr_accessor :data_source_id
 
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
```
Comment (gigxz) [inline drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb:14]: Is this a good idea? The job takes quite a lot to run.

In order to run it asap on the long-running queue, then, I would do `Hmis::MigrateAssessmentsJob.set(priority: 0).perform_later(...)`

Comment (eanders) [reply drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb:14]: Definitely this should be on the long-running queue, so yeah, that's correct.  You probably don't need to set priority unless there's a bunch of stuff in the queue you want to skip.

**Review by eanders — APPROVED** [review]
Looks reasonable to me!

File: drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb (hunk 2 above)
Comment (gigxz) [reply drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb:14]: Ok. Thanks

**Pull request merged**